res/skins/Shade/style-linux.qss

```diff
@@ -0,0 +1,10 @@
+/* hack around text getting cut off with scaled checkbox on Macs with Retina screens */
```

Comment:
This comment is confusing in `style-linux.qss`

Reply:
whoopsy..
anything else slipped through?